gtk/src/default/gtk-3.20/_common.scss

```diff
@@ -2820,7 +2828,7 @@ switch {
     color: $selected_fg_color;
     border-color: $checkradio_borders_color;
     background-color: $checkradio_bg_color;
-    text-shadow: 0 1px transparentize($checkradio_borders_color, 0.5),
+    text-shadow: 0 1px transparentize($selected_borders_color, 0.5),
```

Comment:
do we lose our check/radio color customization?

Reply:
Oh crap, this is an issue! Thanks - need to fix this

Reply:
Double checked, no this does not affect us, we do not have the on/off logo. So we could leave this as upstream - okay for you?